cvat-ui/src/actions/models-actions.ts

```diff
@@ -485,9 +485,10 @@ ThunkAction<Promise<void>, {}, {}, AnyAction> {
                 );
 
                 parse(response.run)
-                    .filter((inferenceMeta: InferenceMeta): boolean => inferenceMeta.active)
                     .forEach((inferenceMeta: InferenceMeta): void => {
-                        subscribe('auto_annotation/check', inferenceMeta, dispatch);
+                        if (inferenceMeta.active) {
```

Comment:
I consider the previous version more 'functional programming style'.
Don't think you so?

Reply:
That was all about performance. But if you are sure that it is OK, it can be closed.